lumicks/pylake/force_calibration/power_spectrum.py

```diff
@@ -156,3 +157,105 @@ def plot(self, **kwargs):
             plt.title(f"Blocked Power spectrum (N={self.num_points_per_block})")
         else:
             plt.title("Power spectrum")
+
+
+def identify_peaks(
```

Comment:
Something to think about:

If this is public API, it would be nice to give it a name that ties it more specifically to power spectral calibration. Perhaps something like `identify_calibration_nuisance_peaks`.

Either that, or we add a method to `PowerSpectrum` for doing this and just keep this one private indefinitely. Then it's tied to those type of analyses at least.

The reason I'm saying this, is because the function name `identify_peaks` is rather general and I wonder if we'll get a clash with a future case where we might wanna use it.

Reply:
I wondered whether or not to make it a method of `PowerSpectrum`, and the reason I opted against it is because it doesn't work for every instance (but only for `num_points_per_block ==1`). Not sure if you see this differently, but I for sure have no objections to renaming this function

Reply:
I think it's ok to be a member function and still raise when `num_points_per_block` isn't `1`.